admin_manual/installation/apps_management_installation.rst

```diff
@@ -40,10 +40,9 @@ supported; they are maintained by trusted developers, and are under active
 development. Only **Official** and **Approved** apps are linked on the Apps 
 page by default.
 
-Click the app name to view a description of the app and any of the app settings in the Application View field.  Clicking the **Enable** button will enable the app.  If the app is not part of the ownCloud installation, it will be downloaded from the app store, installed and enabled. 
+Click the app name to view a description of the app and any of the app settings in the Application View field.  Clicking the **Enable** button will enable the app.  If the app is not part of the ownCloud installation, it will be downloaded from the ownCloud Marketplace, installed and enabled. 
```

Comment:
Maybe extend the text and mention the market app.
Example: "If the app is not part of the ownCloud installation, you can use the market app (link to app detail page on marketplace; will be uploaded today) to manage your apps. For a manual installation of apps go to the ownCloud marketplace (link)"